index.js

```diff
@@ -20,7 +20,11 @@ var DEFAULT_OPTIONS = {
   babel: {
     presets: [
       'react',
-      'es2015',
+      [ "env", {
+        "targets": {
+          "node": "current",
+        },
+      }],
```

Comment:
Can you format this to match the rest of the file: newline inside the array, use single quotes.